pom.xml

```diff
@@ -7,6 +7,19 @@
     <groupId>org.example</groupId>
     <artifactId>cardGamePeterPan</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>14</source>
+                    <target>14</target>
+                    <compilerArgs>--enable-preview</compilerArgs>
```

Comment:
whi is it requried?

Reply:
I see it as connected to possibility of creating multi-line Strings.